TMS_Tests/Tests/AddRemoveElementsTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OpenQA.Selenium;
+
+namespace TMS_Tests.Tests
+{
+    internal class AddRemoveElementsTest : BaseTest
+    {
+        [Test]
+        public void AddRemoveElements()
+        {
+            Driver.Navigate().GoToUrl("http://the-internet.herokuapp.com/add_remove_elements/");
+            Driver.FindElement(By.XPath("//button[text()='Add Element']")).Click();
+            Driver.FindElement(By.XPath("//button[text()='Add Element']")).Click();
```

Comment:
You use this more than ones. Better create veriable
Example:  var addElementDutton = Driver.FindElement(By.XPath("//button[text()='Add Element']"));
addElementDutton.Click();
addElementDutton.Click();

Reply:
Updated